coding-concepts.js

```diff
@@ -6,54 +6,67 @@
 // Then, uncomment the console.log() and verify the output. Briefly explain why your initial answer was correct or incorrect.
 
 // --------------------INSTRUCTOR EXAMPLE: What will this log?
-const colors = ["tangerine", "magenta", "lilac", "daffodil"]
-// console.log(colors.push("indigo"))
+//const colors = ["tangerine", "magenta", "lilac", "daffodil"]
+//console.log(colors.push("indigo"))
 
-// a) Your answer:
+// a) Your answer: 
+ //   5
 // b) Verify and explain:
+ //   The answer is 5 because the .push built-in method adds a value to the end of an array and returns the position of that value. 
 
 
 // --------------------1) What will this log?
 
-const cohort = "LEARN 2022"
-// console.log(cohort.length)
+//const cohort = "LEARN 2022"
+//console.log(cohort.length)
 
 // a) Your answer:
+   // This will log 10.
 // b) Verify and explain:
+ //   I have verified that this logs 10. This is because the .length property returns the length of the string, including spaces. 
 
 
 // --------------------2) What will this log?
 
-const greeting = "Hello World!"
-// console.log(greeting[4])
+//const greeting = "Hello World!"
+//console.log(greeting[4])
 
 // a) Your answer:
+  //  This will log o. 
 // b) Verify and explain:
+ //   I have verified that this logs o. This is because the index [4] returns the index of the fourth value, counting from left to right, starting at zero. 
+
 
 
 // --------------------3) What will this log?
 
-const languages = ["JavaScript", "Ruby", "Python", "C++"]
-const index = 1
-// console.log(languages[index])
+//const languages = ["JavaScript", "Ruby", "Python", "C++"]
+//const index = 1
+//console.log(languages[index])
 
 // a) Your answer:
+   // This will log Ruby. 
 // b) Verify and explain:
+   // I have verified that this logs Ruby. This is because we have identified the first value in the array which is Ruby, with "JavaScript" being the in the 0 index position. 
 
 
 // --------------------4) What will this log?
 
-const weekendDays = ["saturday", "sunday"]
-// console.log(weekendDays.toUpperCase())
+//const weekendDays = ["saturday", "sunday"]
+//console.log(weekendDays.toUpperCase())
 
 // a) Your answer:
+   // This will log SATURDAY, SUNDAY.
 // b) Verify and explain:
+  //  I received an error stating "weekendDays.toUpperCase is not a function". I believe this may be due to the fact that the string is within square brackets. Coded correctly, this string should be modified and returned as all capitalized letters. 
```

Comment:
To utilize .toUpperCase(), it has to be used on a string and not on an array.  To achieve the result `["SATURDAY", "SUNDAY"]`, some form of iteration onto each element in the array would need to be implemented.